### ISSUE TITLE
Error on transaction appears to block ordhook

### ISSUE THREAD

**Issue by GuutBoy** [hirosystems/bitcoin-indexer#267]
I am getting this error
```
ERRO Unable to compute inscription's Satoshi: Unable to process transaction 0xcd5f226e2d82555eeacf2d885e5304774d1e945dd0032b6a349b08a34b4ed320 detected after 829862 iterations. Manual investigation required
```
And suddenly Ordhook is not progressing past block `829861`. It appears to be an error involving this transaction https://mempool.space/tx/cd5f226e2d82555eeacf2d885e5304774d1e945dd0032b6a349b08a34b4ed320.

Is there any way to recover from this?

**Comment by GuutBoy** [comment]
Looks like  #261 fixes this 👍

**Comment by lgalabru** [comment]
This issue was addressed in `v2.1.0`, in the process of being released.
